src/main/java/gtclassic/block/GTBlockMachine.java

```diff
@@ -46,18 +46,10 @@
 
 	String name;
 	String texture;
-	int size = 0;
+	int size;
 
 	public GTBlockMachine(String name, LocaleComp comp) {
-		super(Material.IRON);
-		this.name = name;
```

Comment:
hey @Meduris can you put all the constructor fixes like this into the other PR? they seem to be in both atm

Reply:
that just happened when creating a separate branch for this changes based on the one for the first pull request. Once the first pull request is merged, I'll rebase this branch which should remove those commits

Reply:
👌